packages/realm/rollup.config.mjs

```diff
@@ -92,6 +93,38 @@ export default [
     ],
     external: ["bson", "debug", "react-native"],
   },
+  {
+    input: "src/browser/index.ts",
+    output: {
+      file: mainExport.browser,
+      format: "es",
+      sourcemap: true,
+    },
+    plugins: [
+      nodeResolve({
+        mainFields: ["browser", "module", "main"],
+        resolveOnly: ["@realm/network-transport", "path-browserify"],
+      }),
+      // We need to use `commonjs` because of "path-browserify"
+      commonjs(),
```

Comment:
It seems confusing that we need to use `commonjs()` when doing `format: "es"`, which I think is the opposite of cjs. @kraenhansen can you look into this and suggest either removing it or wording for a comment that explains why this makes sense even when not output cjs?